pkg/doc/verifiable/common.go

```diff
@@ -49,14 +49,14 @@ func (ja JWSAlgorithm) jose() (jose.SignatureAlgorithm, error) {
 	}
 }
 
-// PublicKeyFetcher fetches public key for JWT signing verification based on Issuer ID (possibly DID)
+// PublicKeyFetcher fetches public key for JWT signing verification based on DID
 // and Key ID.
 // If not defined, JWT encoding is not tested.
-type PublicKeyFetcher func(issuerID, keyID string) (interface{}, error)
+type PublicKeyFetcher func(did, keyID string) (interface{}, error)
```

Comment:
I'm a bit confused by this change. Is this a DID specific interface or a generic one for keys?